hobo/script/orm_hobo.py

```diff
@@ -84,11 +65,19 @@ class SensorInfo(BASE):
     """
     __tablename__ = 'sensor_info'
 
+    class SensorTypeEnum(enum.Enum):
```

Comment:
The class uses the import enum. However the remainder of the code uses the Sql Alchemy enums. Why?

Reply:
Using a separate python enum class allows us to access the enum's values in other places like the api/extract scripts. (Also matches the way they set enums in their documentation [here](https://docs.sqlalchemy.org/en/13/core/type_basics.html#sqlalchemy.types.Enum) )

It looks like I could set sqlalchemy enums just with a list instead of the other enum. But then I don't think I would be able to access those enums in the script files, so I'd probably just use literal strings 'egauge', 'database_insertion', etc when setting values for those columns.

Tbh, it might make the code simpler if we just use literal strings instead of accessing values using a class like ```orm_hobo.ErrorLog.PipelineStageEnum.database_insertion```

Any preferences if we'd rather set those values using literal strings or classes?

Reply:
I don't understand. Are you saying the Enum class from python and SqlAlchemy are different? If they are the same, then I don't understand why we can't just stick to one import command.

Reply:
Ok per the call these Enum classes are not identical, therefore both are needed.